<launch>
    <arg name="model"           default="burger"/>
    <arg name="first_tb3"       default="tb3_0"/>
    <arg name="first_tb3_x_pos" default="0.5"/>
    <arg name="first_tb3_y_pos" default="1.7"/>
    <arg name="first_tb3_z_pos" default="0.0"/>
    <arg name="first_tb3_yaw"   default="0.0"/>

    <!-- gmapping -->
    <include file="$(find exam)/launch/gmapping.launch"/>

    
    <group ns = "$(arg first_tb3)">
        <!-- MOVE BASE SERVER -->
        <include file="$(find turtlebot3_navigation)/launch/move_base.launch">
            <arg name="topic_robot"         value="$(arg first_tb3)"/>
            <arg name="model"               value="$(arg model)" />
            <arg name="move_forward_only"   value="false"/>
        </include>
    </group>

    <node pkg="exam" type="exploration" name="exploration" output="screen"/>

</launch>